test/foundry/PriceFeedDispatcher.t.sol

```diff
@@ -87,109 +101,56 @@ contract PriceFeedDispatcherTest is IPriceFeedDispatcherEvent, PriceFeedDispatch
-        _priceFeedDispatcher.setChainlinkPriceFeedV3(newChainlinkPriceFeedV3);
-        assertEq(address(_priceFeedDispatcher.getChainlinkPriceFeedV3()), address(newChainlinkPriceFeedV3));
-
-        vm.expectRevert(bytes("PFD_CNC"));
-        _priceFeedDispatcher.setChainlinkPriceFeedV3(ChainlinkPriceFeedV3(address(0)));
-    }
-
-    function test_cannot_setChainlinkPriceFeedV3_by_nonOwnerAddress() public {
-        ChainlinkPriceFeedV3 newChainlinkPriceFeedV3 = _create_ChainlinkPriceFeedV3();
-        vm.expectRevert(bytes("Ownable: caller is not the owner"));
-        vm.prank(nonOwnerAddress);
-        _priceFeedDispatcher.setChainlinkPriceFeedV3(newChainlinkPriceFeedV3);
+        assertEq(_priceFeedDispatcher.isToUseUniswapV3PriceFeed(), false);
+        _dispatchPrice_and_assertEq_getDispatchedPrice(_chainlinkPrice);
     }
 
-    function test_setUniswapV3PriceFeed() public {
-        _expect_emit_event_from_PriceFeedDispatcher();
-        emit UniswapV3PriceFeedUpdated(UniswapV3PriceFeed(address(0)));
-        _priceFeedDispatcher.setUniswapV3PriceFeed(UniswapV3PriceFeed(address(0)));
-        assertEq(address(_priceFeedDispatcher.getUniswapV3PriceFeed()), address(0));
-
-        UniswapV3PriceFeed newUniswapV3PriceFeed = _create_uniswapV3PriceFeed();
-        _expect_emit_event_from_PriceFeedDispatcher();
-        emit UniswapV3PriceFeedUpdated(newUniswapV3PriceFeed);
-        _priceFeedDispatcher.setUniswapV3PriceFeed(newUniswapV3PriceFeed);
-        assertEq(address(_priceFeedDispatcher.getUniswapV3PriceFeed()), address(newUniswapV3PriceFeed));
-
-        vm.expectRevert(bytes("PFD_UECOU"));
-        _priceFeedDispatcher.setUniswapV3PriceFeed(UniswapV3PriceFeed(address(1)));
-
+    function test_dispatchPrice_isToUseUniswapV3PriceFeed_when__status_is_already_UniswapV3() public {
         _priceFeedDispatcher.setPriceFeedStatus(Status.UniswapV3);
-        vm.expectRevert(bytes("PFD_UNS"));
-        _priceFeedDispatcher.setUniswapV3PriceFeed(UniswapV3PriceFeed(address(0)));
-    }
-
-    function test_cannot_setUniswapV3PriceFeed_by_nonOwnerAddress() public {
-        UniswapV3PriceFeed newUniswapV3PriceFeed = _create_uniswapV3PriceFeed();
-        vm.expectRevert(bytes("Ownable: caller is not the owner"));
-        vm.prank(nonOwnerAddress);
-        _priceFeedDispatcher.setUniswapV3PriceFeed(newUniswapV3PriceFeed);
-    }
 
-    function test_dispatchPrice_not__isToUseUniswapV3PriceFeed() public {
-        assertEq(uint256(_priceFeedDispatcher.getStatus()), uint256(Status.Chainlink));
-        _dispatchPrice_and_assertEq_getDispatchedPrice(_chainlinkPrice);
+        _dispatchPrice_and_assertEq_getDispatchedPrice(_uniswapPrice);
+        assertEq(uint256(_priceFeedDispatcher.getStatus()), uint256(Status.UniswapV3));
     }
 
-    function test_dispatchPrice__isToUseUniswapV3PriceFeed_when__chainlinkPriceFeed_isTimedOut() public {
+    function test_dispatchPrice_isToUseUniswapV3PriceFeed_when__chainlinkPriceFeed_isTimedOut() public {
```

Comment:
```suggestion
    function test_dispatchPrice_isToUseUniswapV3PriceFeed_when_different__chainlinkPriceFeed_isTimedOut public {
```
should we name it somewhat like this? because it contains two _isTimeOut scenarios